app/controllers/aspect_memberships_controller.rb

```diff
@@ -68,7 +68,7 @@ def create
   end
 
   rescue_from ActiveRecord::StatementInvalid do
-    render :text => "Duplicate record rejected.", :status => 400
+    render :text => I18n.t('aspect_memberships.destroy.invalid_statement'), :status => 400
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.